api/schema/application_aws_resources_resource.go

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Illumio, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var (
+	applicationAwsResourcesResource = Resource{
+		TypeName: "application_aws_resources",
+		Schema: resource_schema.Schema{
+			Version:     1,
+			Description: "Manages a set of AWS resources belonging to a single AWS account that are associated with a CloudSecure application.",
+			Attributes: map[string]resource_schema.Attribute{
+				IDFieldName: idAttribute,
+				"application_id": StringResourceAttributeWithMode{
+					StringAttribute: resource_schema.StringAttribute{
+						Description: "ID of the CloudSecure application.",
+						Required:    true,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.RequiresReplace(),
+						},
+					},
+					attributeWithMode: attributeWithMode{
+						Mode: ImmutableAttributeMode,
+					},
+				},
+				"account_id": StringResourceAttributeWithMode{
+					StringAttribute: resource_schema.StringAttribute{
+						Description: "ID of the AWS account the AWS resources belong to.",
+						Required:    true,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.RequiresReplace(),
+						},
+					},
+					attributeWithMode: attributeWithMode{
+						Mode: ImmutableAttributeMode,
+					},
+				},
+				"aws_arns": resource_schema.ListAttribute{
```

Comment:
Since you split the resource into one per CSP, and the CSP name is included in the resource's name, the `aws_` prefix for attributes is redundant.
```suggestion
				"arns": resource_schema.ListAttribute{
```
Same for all the other attributes below.

Reply:
This comment has not been addressed. All the other attributes below still have the aws_ prefix. Remove that for all attributes.